app/javascript/crayons/Modal/Modal.jsx

```diff
@@ -54,7 +54,7 @@ export const Modal = ({
         aria-label="modal"
         className="crayons-modal__box"
       >
-        {title.length > 0 && title && (
+        {title && title.length > 0 && (
```

Comment:
Out of curiosity (and I know it's not related to this PR)... Why do we check for `title.length` as well? cc @nickytonline

Reply:
seems a bit strange since an empty string would be falsey anyway 🤔

Reply:
Refactor (blocking): Checking `title` is good enough for a check. It will validate for  `""`, `null` or `undefined`

```suggestion
        {title && (
```

Reply:
Yeah I thought it was weird but I was more concerned with the error of checking length before we verify it exists. I just thought there must have been some wacky reason ya'll were checking for it. Removed!